Library/Homebrew/dev-cmd/update-license-data.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require "commands"
+require "cli/parser"
+require "open-uri"
+require "json"
+
+module Homebrew
+  module_function
+  SPDX_FOLDER_PATH = (HOMEBREW_LIBRARY_PATH/"data").freeze
+  FILE_NAME = "spdx.json".freeze
+  NEW_FILE_NAME = "spdx_new.json".freeze
```

Comment:
Just download the new file straight over the old one and use `git diff` to check for differences in CI e.g. https://github.com/Homebrew/brew/blob/0dbbefe33f39400045fec646595670e1dfbed769/.github/workflows/tests.yml#L90-L91 or put this in the command with `--fail-if-changed`